csrc/python_frontend/python_bindings.cpp

```diff
@@ -2726,6 +2735,48 @@ void initNvFuserPythonBindings(PyObject* module) {
   py::class_<FusionDefinition::SchedOperators> nvf_sched(
       fusion_def, "SchedOperators");
   nvf_sched.def(py::init<FusionDefinition*>());
+  //! experimental API for multidevice support
+  nvf_sched.def(
+      "_create_device_mesh",
+      [](FusionDefinition::SchedOperators& self, std::vector<int64_t> devices) {
+        FUSER_PERF_SCOPE("SchedOperators._create_device_mesh");
+        return DeviceMesh(devices);
+      },
+      py::arg("devices"),
+      py::return_value_policy::reference);
+  //! experimental API for multidevice support
+  nvf_sched.def(
+      "_set_device_mesh",
+      [](FusionDefinition::SchedOperators& self, Tensor arg, DeviceMesh mesh) {
```

Comment:
Does the name `arg` mean anything special? For example, does it have to be a fusion input? If not, we could call it `tensor` or `t` for conciseness and less confusion.

Reply:
That's just a boilerplate variable name for pybind functions, since all pybind args is declared with py::arg.

I actually don't know if there's a convention here. But I agree that tensor is probably more specific and a better choice. I'll update it for added functions here.